docs/sources/shared/flow/reference/components/loki-server-grpc.md

```diff
@@ -0,0 +1,22 @@
+---
+aliases:
+- /docs/agent/shared/flow/reference/components/loki-server-grpc/
+headless: true
+---
+
+The `grpc` block configures the gRPC server.
+
+The following arguments can be used to configure the `grpc` block. Any omitted
+fields take their default values.
+
+ Name                            | Type       | Description                                                                                                          | Default      | Required
+---------------------------------|------------|----------------------------------------------------------------------------------------------------------------------|--------------|----------
+ `listen_address`                | `string`   | Network address on which the server will listen for new connections. Defaults to accepting all incoming connections. | `""`         | no
```

Comment:
Same here, does the description mean that the default is `0.0.0.0`? Or is it implied to be 0.0.0.0 by the weaveworks.Server code?